src/components/resource-selector/media-library/media-library-file-dropzone.yml

```diff
@@ -0,0 +1,4 @@
+mediaLibraryFileDropzone:
+  fileWillBeAddedToMediaLibrary:
+    en: This file will be added to the Media library
```

Comment:
Small "m" in this case.

Reply:
Corrected (also in "Search in the Media library"), I guess it is not enough of a proper name to justify uppercasing.